processors/base/helpers/__init__.py

```diff
@@ -163,6 +164,19 @@ def validate_identifier(identifier):
             return int(numbers) != 0
 
 
+def validate_remote_url(url):
+    """"URLs without domain and scheme or with a file scheme are invalid"""
+    is_valid_url = False
+    try:
+        parsed_url = urlparse.urlparse(url)
+        if parsed_url.scheme and parsed_url.netloc:
+            is_valid_url = parsed_url.scheme != 'file'
```

Comment:
To improve legibility, please add paranthesis here (i.e. `is_valid_url = (parsed_url.scheme != 'file')`)